libs/ui-lib/lib/common/components/ui/formik/validationSchemas.ts

```diff
@@ -38,8 +38,8 @@ const CLUSTER_NAME_VALID_CHARS_REGEX = /^[a-z0-9-]*$/;
 const SSH_PUBLIC_KEY_REGEX =
   /^(ssh-rsa|ssh-ed25519|ecdsa-[-a-z0-9]*) AAAA[0-9A-Za-z+/]+[=]{0,3}( .+)?$/;
 const DNS_NAME_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/;
-const BASE_DOMAIN_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*)$/;
-const DNS_NAME_REGEX_OCM = /^([a-z0-9]+(-[a-z0-9]+)*[.])+[a-z]{2,}$/;
+const BASE_DOMAIN_REGEX = /^[a-z\d][\-]*[a-z\d]+$/;
```

Comment:
Is the error message we show aligned with the new Regular Expression?

From your screenshot, you entered a value which didn't have less than 2 characters after the dot (and probably an "at least" is currently missing), and it doesn't end with a special character.

So if presented with the error, users wouldn't know how to fix it.

Reply:
@celdrake you suggest to change to:
" DNS base domains must have at least two characters after the dot and must not end in a special character."

Reply:
I'm not sure I understand the context of your remark here. What's the problem with BASE_DOMAIN_REGEX?

Reply:
it's not related to the line of code, but from the screenshot the message doesn't seem totally accurate.
It can be ignored, it does not affect the MR.